index.js

```diff
@@ -927,7 +927,7 @@ Command.prototype._checkForMissingMandatoryOptions = function() {
  * void of these options.
  *
  * @param {Array} argv
- * @return {Array}
+ * @return {{args: Array, unknown: Array}}
```

Comment:
https://github.com/tj/commander.js/pull/1132/files#diff-168726dbe96b3ce427e7fedce31bb0bcR966

`return this.optionMissingArgument(option);`

Need to mention this 'return'?

Reply:
`optionMissingArgument` calls `process.exit` or throws so the `return` never actually returns anything (the remaining warning is about returning void).

 I think the correct fix is to delete the `return` in this line. I didn't do that this time because I was only changing comments and not code, but I could do that again?

Reply:
FYI: this is fixed in #1138